src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -70,6 +72,11 @@ function disableFormEditInstructor(number) {
 function showNewInstructorForm() {
     $('#panelAddInstructor').show();
     $('#btnShowNewInstructorForm').hide();
+    if (hasClickedAddInstructorBefore === false) {
```

Comment:
Changed it to bind on `document.ready`, the implementation here was due to the idea that binding would not work on hidden elements, however this appears not to be the case.